Dockerfile

```diff
@@ -161,13 +161,16 @@ ENV PATH="/root/stp/build:${PATH}"
 ENV LD_LIBRARY_PATH="/root/stp/deps/cadical/build:/root/stp/deps/cadiback/:$LD_LIBRARY_PATH"
 
 # Build Yosys.
-WORKDIR /root
-RUN source /root/dependencies.sh \
-  && mkdir yosys && cd yosys \
-  && wget -qO- https://github.com/YosysHQ/yosys/archive/$YOSYS_COMMIT_HASH.tar.gz | tar xz --strip-components=1 \
-  && PREFIX="/root/.local" CPLUS_INCLUDE_PATH="/usr/include/tcl8.6/:$CPLUS_INCLUDE_PATH" make config-gcc \
-  && PREFIX="/root/.local" CPLUS_INCLUDE_PATH="/usr/include/tcl8.6/:$CPLUS_INCLUDE_PATH" make -j ${MAKE_JOBS} install \
-  && rm -rf /root/yosys
+WORKDIR /root 
+RUN cd /root \
+&& git clone https://github.com/YosysHQ/yosys.git \
```

Comment:
Can you add a comment here saying that Yosys depends on a submodule and so we cannot download an archive of it? Furthermore, say that, once https://github.com/dear-github/dear-github/issues/214 is resolved, we could switch back to downloading an archive.